Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -168,13 +159,91 @@ public struct SymbolRole: OptionSet {
   }
 }
 
+public enum IndexSymbolKindEnum: Int {
```

Comment:
I'm not sure if we actually want an enum, or to mimic an enum with `RawRepresentable` like `SymolRole`.

If we make it an enum, it's cleaner, but we cannot directly use the values from the C enum and risk things getting out of sync.

I've included both approaches here. Let me know which one I should remove

Reply:
It should be an enum, but it should use a switch to convert between them, not use the raw value directly.  That way we are using the C value by name.

Reply:
By the way, after that change it won't need to be an `Int` raw value enum anymore